packages/studio/src/components/EditStreamScopeModal.tsx

```diff
@@ -0,0 +1,98 @@
+import { useCallback, useMemo, useState } from "react";
+import DialogModal from "./common/DialogModal";
+import StreamScopeInput from "./StreamScopeInput";
+import { ResponseType, StreamScope } from "@yext/studio-plugin";
+import useStudioStore from "../store/useStudioStore";
+import { isEqual } from "lodash";
```

Comment:
For improved tree-shaking and bundle size, we should use https://www.npmjs.com/package/lodash.isequal instead of importing all of lodash.

Reply:
think this advice is more up to date! https://lodash.com/per-method-packages
I ended up following this in the bundling PR and it did make a difference! didn't try the completely separate packages but I just took their word for it that the per method packages are outdated

Reply:
gotcha! so I should leave the import it as is